docs/src/trace-viewer.md

```diff
@@ -234,7 +234,8 @@ context.tracing().stop(new Tracing.StopOptions()
 ```
 
 ```csharp
-await using var browser = playwright.Chromium.LaunchAsync();
```

Comment:
we have this kind of bug in a few more places, do you mind fixing them too?

<img width="543" alt="image" src="https://github.com/microsoft/playwright/assets/17984549/614ce884-aaf6-462f-bcb4-c6dd48fee95a">

Reply:
done :)